gen_pack.ts

```diff
@@ -128,7 +128,7 @@ async function getMetadata(opt: ModOptions): Promise<Metadata> {
     return JSON.parse(metadataJson);
   } else {
     return {
-      nightMode: !!opt.nightMode,
+      nightModeAvailable: !!opt.nightModeAvailable,
```

Comment:
a revert, le vrai fix est `"nightMode": !!this.story?.nightModeAvailable,` dans extract, ça permet aussi de garder la compatibilité avec les metadata.json existant.
pour résumer le bug : dans les metadata.json on utilise "nightMode" mais l'outil d'extraction générait des metadata avec "nightModeAvailable"